File: Morning-snacks/PerfectNumbersInARange.java
public class PerfectNumbersInARange{

	public static void main(String... args){

		for(int count = 1; count <= 1000; count++){

			int sumOfFactors = 0;

			for(int index = 1; index <= count / 2; index++){

				if (count % index == 0){

					sumOfFactors += index;

				}
			}

			if (count == sumOfFactors){

				System.out.println(count);

			}
		}


	}


}

File: Morning-snacks/PerfectNumbersInARange.js
for(let count = 1; count <= 1000; count++){

	let sumOfFactors = 0;

	for(let index = 1; index <= Math.floor(count / 2); index++){

		if (count % index == 0){

			sumOfFactors += index;

		}

		if (count == sumOfFactors){

			console.log(count);

		}

	}

}